portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,24 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+
```

Comment:
thought: Should we remove newline on 19?

Reply:
Well, it looks good after removing it.
So I'm removing it :smile:

Reply:
:D sounds good. Marking this as resolved.